addons/event_booth_sale/static/src/js/event_booth_configurator_widget.js

```diff
@@ -0,0 +1,148 @@
+odoo.define('event_booth_sale.product_configurator', function (require) {
+const ProductConfiguratorWidget = require('sale.product_configurator');
+
+/**
+ * Extension of the ProductConfiguratorWidget to support event booth configuration.
+ * It opens when an event booth product_product is set.
+ *
+ * The event booth information include:
+ * - event_id
+ * - event_booth_id
+ *
+ */
+ProductConfiguratorWidget.include({
+
+    //--------------------------------------------------------------------------
+    // Overrides
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {boolean}
+     *
+     * @override
+     * @private
+     */
+    _isConfigurableLine: function () {
+        return this.recordData.is_event_booth || this._super.apply(this, arguments);
+    },
+
+    /**
+     * Opens the event booth configurator in 'edit' mode
+     *
+     * @override
+     * @private
+     */
+    _onEditLineConfiguration: function () {
+        if (this.recordData.is_event_booth) {
+            let defaultValues = {
```

Comment:
I think you can (and should?) go with "const" here.
I learnt recently that defining a const object still lets you modify its keys (yay, JS...).